left.php

```diff
@@ -35,20 +35,14 @@
 
 echo "<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Frameset//EN'
        'http://www.w3.org/TR/html4/frameset.dtd'>";
-echo "\n<html><head><title>".sprintf(__('%1$s - %2$s'), "GLPI", __('Tree view', 'treeview'))."</title>";
-echo "<meta http-equiv='Content-Type' content='text/html; charset=utf-8'>";
-// Send extra expires header if configured
-echo "<meta http-equiv='Expires' content='Fri, Jun 12 1981 08:20:00 GMT'>\n";
-echo "<meta http-equiv='Pragma' content='no-cache'>\n";
-echo "<meta http-equiv='Cache-Control' content='no-cache'>\n";
-echo "<link rel='stylesheet' type='text/css' href='$treeview_url/css/treeview.css' type='text/css'>\n";
-echo "<script type='text/javascript' src='$treeview_url/js/treeview.js'>
-      </script></head>\n";
+echo "\n<html>";
+
+Html::includeHeader("TreeView");
```

Comment:
@stonebuzz  `Html::includeHeader()`  will output `<!DOCTYPE html>` and `<html>` tags and will produce an invalid DOM.

Reply:
done by removing duplicate HTML tag (from plugin)